### PR TITLE
Fix/AUT-741/Rich passage handlers for taoQtiItem and taoQtiTestPreviewer 

### PR DESCRIPTION
**Pull request by bziondik**
Related to: https://oat-sa.atlassian.net/browse/AUT-741

Create handlers for taoQtiItem (adding passage in ItemAuthoring) and taoQtiTestPreviewer (item preview with rich passage)
Update selectors for Style Editor: use prefix `body div.qti-item`
Set hashClass to `.item-body`

### CODE REVIEW

File: scripts/install/RegisterItemDataHandler.php [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+                        'category' => 'handler',
+                        'active' => true,
+                        'tags' => []
+                    ]
+                ]
+            ]);
+
+        return \common_report_Report::createSuccess('Rich passage handler registered for taoQtiTestPreviewer');
+    }
+}
```
Comment (bartlomiejmarszal) [inline scripts/install/RegisterItemDataHandler.php:53]:
```suggestion
}

```

File: scripts/install/RegisterXinludeHandler.php [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+                        'category' => 'handler',
+                        'active' => true,
+                        'tags' => []
+                    ]
+                ]
+            ]);
+
+        return \common_report_Report::createSuccess('Rich passage handler registered for taoQtiItem');
+    }
+}
```
Comment (bartlomiejmarszal) [inline scripts/install/RegisterXinludeHandler.php:53]:
```suggestion
}

```

**Review by bartlomiejmarszal — COMMENTED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Review by bartlomiejmarszal — APPROVED** [review]
BE side - LGTM

**Review by AnastasiaArcadia — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not master

**Review by Silvia-loza — APPROVED** [review]
FE. looks good to me and it all wks as expected.
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [X] New code is respecting code style rules
- [X] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [X] Feature is working correctly on my local machine (if applicable)
- [X] Acceptance criteria are respected
- [X] Pull request title and description are meaningful

**Pull request merged**